packages/main/src/queue/combineHandlers.ts

```diff
@@ -125,11 +125,13 @@ function onFailure<HM extends Handlers>(
   error: JobFailure<HM>,
   _stats: { elapsed: number }
 ) => Promise<void> {
-  return async (_taskId, { cause, task }) => {
-    const failureHandler = handlers[task.type].failure
-    if (failureHandler) {
-      await failureHandler(cause, task.params)
-    }
+  return async (_taskId, error) => {
+    console.error(error)
```

Comment:
I made a change here because this code did not work as I thought it did, and I wanted to have a descriptive message in the console when a queued task fails.